pom.xml

```diff
@@ -4,100 +4,202 @@
     <artifactId>xoai</artifactId>
     <name>XOAI : OAI-PMH Java Toolkit</name>
     <version>3.2.11-SNAPSHOT</version>
+    <url>https://github.com/dspace/xoai</url>
 
-    <parent>
-        <groupId>org.sonatype.oss</groupId>
-        <artifactId>oss-parent</artifactId>
-        <version>9</version>
-    </parent>
+    <prerequisites>
```

Comment:
We should use the enforcer plugin for this.  I'm told that <prerequisites> is for Maven plugin projects only.

Reply:
Good point, I forgot about this. Fixed in latest commit

Reply:
Good point, I forgot about this. Fixed in latest commit